packages/react-server/src/render/render.tsx

```diff
@@ -39,7 +38,7 @@ type Options = Pick<
  */
 export function createRender(render: RenderFunction, options: Options = {}) {
   return async function renderFunction(ctx: Context) {
-    const logger = getLogger(ctx) || console;
+    const logger = ctx.state.logger || console;
```

Comment:
Why not continue to use `getLogger` rather than storing directly on state?

Reply:
Ou, true. Also `LOGGER = Symbol('logger');` so better to keep the `getLogger`